cypress/fixtures/object-templates.ts

```diff
@@ -0,0 +1,55 @@
+export const StudyTemplate = {
```

Comment:
I am thinking should we have the var names included `Template` here since we may get confused with our current `templates` ?

Reply:
I've renamed these test objects as `TestStudyObject`, `TestDACObject`...

Reply:
They sound good to me 👍